scripts/quarantine_integration.yaml

```diff
@@ -521,13 +521,24 @@
   comment: "Configurations excluded to limit resources usage in integration builds"
 
 - scenarios:
-    - sample.benchmark.coremark_heap
+    - sample.benchmark.coremark.flash_and_run
   platforms:
+    - nrf52dk/nrf52832
```

Comment:
Shouldn't we exclude here more platforms? I would leave just one as we check proper working in  sample.benchmark.coremark. Or we could leave all tests here and remove more platforms from  sample.benchmark.coremark

Reply:
I guess the second solution is better as we will have harness then.

Reply:
the CI execution for the harness configuration hasn't changed

Reply:
Well, I think the default configuration should be tested at least as well as an alternative configuration ("flash_and_run"). Don't you think that?

Reply:
Well, most users would probably work in 90% of cases with the default configuration so I would assume that this last configuration variant that should be broken.

The flash_and_run is a special case of runtime configuration that is also extensively tested as it is the only configuration that allows us to have runtime verification (the remaining test configurations are build-only).

TLDR: I would keep the increased integration test scope for the default configuration.

Reply:
Ok, lets leave it then as it is :)